hydra-node/test/Hydra/Chain/Direct/Contract/Contest.hs

```diff
@@ -223,13 +223,13 @@ genContestMutation
     , _utxo
     ) =
     oneof
-      [ SomeMutation Nothing MutateSignatureButNotSnapshotNumber . ChangeHeadRedeemer <$> do
+      [ SomeMutation (Just "signature verification failed") MutateSignatureButNotSnapshotNumber . ChangeHeadRedeemer <$> do
```

Comment:
This commit and the next should not be in this PR, right? There are no error codes used in this one and I think you might have mixed this up with https://github.com/input-output-hk/hydra/pull/719?

Reply:
@ffakenz Seems like these came from a rebase problem. Can you please just remove the commits which ended up wrong here (it's useless history).

Reply:
I did this now and updated your `mutations/explicit_expected_errors_2` branch with these two commits (squashed)

Reply:
Yeah, my bad had an inconvenience yesterday and by mistake, force-pushed the commits into the wrong branch.

I'll squash these commits.